### ISSUE TITLE
[Accessibility] Insufficient color contrast on <del> tag

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#59748]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

There are multiple instances of legacy CSS styles that set the opacity for sale pricing using the `<del>` tag to .5, which results in a WCAG failure for color contrast, even when the text is black (#000), this results in a contrast ratio of 3.3:1.

<img width="1251" height="772" alt="Screenshot showing sale price rendering as #8E8E8E with a contrast ration of 3.3:1." src="https://github.com/user-attachments/assets/c6410f99-d246-4dbc-b93c-1eac1d304057" />

This issue arose during an extension audit (Product Recommendations) on a site using the Storefront theme. It's not visible in all themes or block-based setups.

@PanosSynetos did some digging and said:

> I don't see this opacity only in Storefront theme. I see it in WooCommerce's core too.
>
> I suggest you fire up an issue in Core, as I see several `del` with opacity: 0.5
>
> See [https://github.com/search?q=repo%3Awoocommerce%2Fwoocommerce%20opacity%3A%200.5%3B&type=code](https://www.google.com/url?q=https://github.com/search?q%3Drepo%253Awoocommerce%252Fwoocommerce%2520opacity%253A%25200.5%253B%26type%3Dcode&sa=D&source=editors&ust=1752766222709749&usg=AOvVaw2wFvv6dGgv864Y8pseO0Ue)
>
> [https://github.com/woocommerce/woocommerce/blob/1a5c8d69e8447fc1a0cff1333b1bdaa8a419f218/plugins/woocommerce/client/legacy/css/woocommerce.scss#L137](https://www.google.com/url?q=https://github.com/woocommerce/woocommerce/blob/1a5c8d69e8447fc1a0cff1333b1bdaa8a419f218/plugins/woocommerce/client/legacy/css/woocommerce.scss%23L137&sa=D&source=editors&ust=1752766222709791&usg=AOvVaw06Zl2gYwovmfZWvkywCzn2)

### Expected behavior

The opacity of sale prices/`<del>` should not be 0.5. It should have a contrast ratio of at least 4.5:1.

It is recommended that the opacity for these elements be set to 0.7, which will pass contrast requirements for both AA and AAA.

### Actual behavior

The current contrast ratio is 3.3:1.

### Steps to reproduce

Unfortunately I don't have detailed steps for replicating this as I am not sure if the site we tested on is supposed to be public. Please see the URLs linked above with where the contrast is set in the code.

Per @PanosSynetos, this issue will be apparent on a site if it is in an area that uses the "legacy" way of displaying products. It can be seen with product recommendations activated. If you need more detailed steps, hopefully he can help tell you how to replicate it.

### WordPress Environment

Unknown, as we only have front-end access to the site. 

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by amberhinds** [comment]
cc @jamesckemp @vikrampm1 @jeffpaul

**Comment by nerrad** [comment]
Yea the defaults should be improved if that's the issue here. However, it's also a common thing overridden by themes.

**Comment by sunyatasattva** [comment]
I created a PR on both [Core](https://github.com/woocommerce/woocommerce/pull/60275) and [Storefront](https://github.com/woocommerce/storefront/pull/2198). Note that the defaults were set only on legacy themes, with blocks we don't really change the opacity of the original price when on sale. Still, while I was at it, I fixed it. I don't think it's worth keeping Product Recommendation from being compliant as it doesn't seem to have anything to do with them and all with the theme. cc @PanosSynetos
